test/browser/stub-xhr.js

```diff
@@ -0,0 +1,525 @@
+// SOURCE: https://github.com/pretenderjs/FakeXMLHttpRequest
```

Comment:
I would strongly advise not to rely on partially extracted code from sinon, we will loose the possibility of easy maintenance. I would rather use the `sinon` as it will be much easier to get any future updates as well as other sinon features if this is going to be needed in future.

Reply:
Yea if you guys are up for adding sinon then I'll happily add it in. Do you think it should be added as a dev dependency, and sourced from `node_modules`, or the dist copied in like has been done with mocha and chai? Same question for the `fetch-mock` too actually.

My guess is the copying was done initially in in order to prevent breaks from node_modules being reorganized, or distribution changes since inclusion is being done in a script tag.

Reply:
For now I would "follow the pattern" so do the same as you added `fetch-mock` unless I'm not aware of something but I don't see a reason to copy the sinon into new folder are you ok with this @mwear ?. I think this template should be refactored anyway to be "dynamic" but this is another issue and not for this PR :)